schema/validator.go

```diff
@@ -54,6 +55,28 @@ func (v Validator) Validate(src io.Reader) error {
 			"schema %s: unable to validate", v)
 	}
 
+	if v == MediaTypeManifest {
```

Comment:
This is not part of the JSON Schema validation. Your new logic should live in `manifest.validate` in `image/manifest.go`.